assets/src/blocks/Articles/ArticlesBlock.js

```diff
@@ -169,7 +169,7 @@ export class ArticlesBlock {
         }
 
         // TO-DO: Check for posts types and posts too...
-        if ((tagsList && tagsList.length === 0) || (postTypesList && postTypesList.length === 0)) {
+        if ((tagsList && tagsList.length === 0) && (postTypesList && postTypesList.length === 0)) {
```

Comment:
I needed to change this locally or the block would never get passed here, not sure if it could cause any issues in some cases. I couldn't find a case but maybe something to keep an eye on. Could be that this is not needed anymore because `populatePostsToken` is moved to `componentDidMount` instead of the constructor (which react was giving warnings about in the browser console.